package.json

```diff
@@ -27,6 +27,7 @@
     "pretest": "npm run compile",
     "prepublish": "npm run compile",
     "test": "node_modules/.bin/mocha -R spec --recursive test/",
+    "test:watch": "node_modules/.bin/mocha -R spec --watch test/",
```

Comment:
Unrelated to the task. Please put in a separate commit.

Reply:
Sure, we'll remove it.